cpp/test/mesh/Domain.xdmf

```diff
@@ -0,0 +1,26 @@
+<Xdmf Version="3.0">
```

Comment:
Can we avoid adding data files? Can it be created by the test rather than added to the repo?

Reply:
I can convert the file to ascii, add it to the source code and have the code write it to `/dev/shm`, then read it back. Would that be better?

Reply:
On second though, that's really just the same thing as adding a file with the ascii content, which is what I do now XD
Please advise!

Reply:
Ok, never mind... I turned on my brain and found a solution.